### PR TITLE
Add documentId and elementId to PTC_onshape_metadata extension schema

### PR DESCRIPTION
**Pull request by gferreira-onshape**
Add elementId and documentId to the schema and example in README.md

### CODE REVIEW

**Review by ron-onshape — APPROVED** [review]

File: extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json [hunk 1]
```diff
@@ -5,6 +5,16 @@
     "description": "glTF extension for specifying IDs and visibility.",
     "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
     "properties": {
+        "documentId" : {
+            "type" : "string",
+            "minLength": 1,
```
Comment (PaulJPremakumar) [inline extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json]: The minimum documentId/elementId size Onshape uses is 24, the specification currently has it set to 1. Is this by design?

File: extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json [hunk 2]
```diff
@@ -5,6 +5,16 @@
...
+            "description" : "An identifier of the document this object relates to."
+        },
+        "elementId" : {
```
Comment (PaulJPremakumar) [inline extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json:13]: The minimum documentId/elementId size Onshape uses is 24, the specification currently has it set to 1. Is this by design?

**Review by PaulJPremakumar — APPROVED** [review]
Looks good.

File: extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json (hunk 1 above)
Comment (gferreira-onshape) [reply extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json]: Nope I just wasn't aware of the minimum, pushing a fix 🙂

**Pull request merged**